sunpy/timeseries/sources/eve.py

```diff
@@ -57,35 +57,59 @@ class ESPTimeSeries(GenericTimeSeries):
 
     _source = 'esp'
 
-    @peek_show
-    def peek(self, title='EVE/ESP Level1', **kwargs):
+    def plot(self, title="EVE/ESP Level 1", **kwargs):
         """
+        Plots the EVE ESP Level 1 timeseries data.
+
         Parameters
         ----------
         title : `str`, optional
-            Plot title.
+            The title of the plot. Defaults to "EVE/ESP Level 1".
         **kwargs : `dict`
```

Comment:
I always forget if this need an optional.

Reply:
Other parts of the code follow the same behavior so I guess we do it here too?